src/entities/ProductDetail.java

```diff
@@ -0,0 +1,56 @@
+package entities;
+
+import enums.MeasureUnit;
+
+import java.util.List;
+
+public class ProductDetail extends BaseEntity<Long>{
+    private Product product;
+    private String name;
+    private Double price;
+    private MeasureUnit measureUnit;
```

Comment:
Hmmm can you think of a better way of doing this? Seems if measure unit changes will be affected as well